java/src/client/game/GameManager.java

```diff
@@ -60,22 +82,24 @@ public void stopPoller() {
      * When the poller finds out the server has a new version it will call this function to update the client game
      * @param cm the new clientModel
      */
-    public void updateClientModels(ClientModel cm){
+    public void updateGameManager(ClientModel cm){
```

Comment:
`GameManager` is this class, and while it's being updated, it's not incredibly obvious what that implies. `updateModel` would be a more appropriate name.